### ISSUE TITLE
Review (This mod is good!)

### ISSUE THREAD

**Issue by Decme** [UltimateBoomer/smoothboot-fabric#29]
I can't find a place to leave a review, and can't find github discussions on this resource so guess it'll go here.

This mod is brilliant, hugely underrated. I'm using 1.18.1 on a Ryzen 1700 together with 23 other fabric client mods.

Without smoothboot, the client takes 45 seconds to load, and pins my 8 core 16 thread CPU at 100% for a lot of that time.
With smoothboot, it loads in 26 seconds, and the CPU is nowhere near 100% utilization.
Basically cuts loadtime in half.

I don't know what magic this is but it seems to work and works really well.

**Comment by pydubz08** [comment]
Totally agree. Before I had Smoothboot installed, loading was nightmare. It was really laggy, some click actions had a big delay. Now it is a breeze :D

**Comment by Boobies** [comment]
Please do not abuse the bug tracker.
